Analysis/DataModel/include/Analysis/HFSecondaryVertex.h

```diff
@@ -50,19 +48,35 @@ DECLARE_SOA_INDEX_COLUMN_FULL(Index1, index1, int, BigTracks, "fIndex1");
 DECLARE_SOA_INDEX_COLUMN_FULL(Index2, index2, int, BigTracks, "fIndex2");
 DECLARE_SOA_INDEX_COLUMN_FULL(Index3, index3, int, BigTracks, "fIndex3");
 DECLARE_SOA_COLUMN(HFflag, hfflag, uint8_t);
+
+DECLARE_SOA_COLUMN(D0ToKPiFlag, d0ToKPiFlag, uint8_t);
+DECLARE_SOA_COLUMN(JpsiToeeFlag, jpsiToeeFlag, uint8_t);
```

Comment:
I guess this should be `JpsiToEEFlag, jpsiToEEFlag` to be consistent with the rest.

Reply:
sure thing, I will change it